lib/message.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Message
+  def initialize
+    @client = Twilio::REST::Client.new(
+        ENV['TWILIO_SID'],
+        ENV['TWILIO_TOKEN']
+      )
+  end
+  def send_message(msg)
+    twilio_number = ENV['TWILIO_NUMBER']
+    client_number = ENV['IRYNA_NUMBER']
+
+    message = @client.account.messages.create(
+      :body => msg,
+      :to => client_number,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)